### ISSUE TITLE
TMC2209 stallguard with NEMA 11 or NEMA 14 Stepper motors? 

### ISSUE THREAD

**Issue by Tupptupp** [teemuatlut/TMCStepper#182]
Has anyone used stallguard with these smaller motor sizes? I haven't been able to find any information online. I'm wondering whether stallguard will still be accurate with smaller back-emf and smaller current draw.

I'm going to use it for a very simple application. Just for closing doors. When the door is closed the stepper will stall and trigger stallguard. I already have this working for a NEMA 17 stepper motor, but space is an issue and I need to downsize the motor.

**Comment by daniel-frenkel** [comment]
Yes, it will work with all motors, just need to set the parameters correctly
